src/react-components/tokens/TokenPageLayout.js

```diff
@@ -8,13 +8,11 @@ import styleUtils from "../styles/style-utils.scss";
 import styles from "./Tokens.scss";
 
 export const TokenPageLayout = ({ children }) => (
-  <StorybookAuthContextProvider>
-    <PageContainer className={styles.tokenContainer}>
-      <Column gap="xl" className={classNames(styles.centerPaddingSides, styleUtils.xlPadding)}>
-        {children}
-      </Column>
-    </PageContainer>
-  </StorybookAuthContextProvider>
+  <PageContainer className={styles.tokenContainer}>
```

Comment:
This looks unrelated. Don't know the context for this, if this is just an unrelated quick change you wanted to put in with this as well that's all good, just confirming its intentional. I usually try to note this in the PR description so whoever is reviewing knows its on purpose.